upgraded UPGRADE.md with experience from upgrading (#868)
shopsys/shopsys: docs/upgrade/UPGRADE-v7.0.0-beta6.md

        - `OrderDataFactory`
        - `OrderItemFacade`
        - `OrderFacade`

>     - add an extension for `OrderItemData` and `OrderItemDataFactory` and register them in `services.yml`

Reviewer: ```suggestion
    - *(low priority)* add an extension for `OrderItemData` and `OrderItemDataFactory` and register them in `services.yml`
```

        - `OrderDataFactory`
        - `OrderItemFacade`
        - `OrderFacade`

>     - add a extension for `OrderItemData` and `OrderItemDataFactory` and register them in `services.yml`

Reviewer 2: I still think it should not be here @boris-brtan

>         - for inspiration follow changes done in [pull request](https://github.com/shopsys/shopsys/pull/715/files)

Reviewer: ```suggestion
```

            - use new methods `getAllEnabledOnDomain` and `getAllEnabledOnCurrentDomain` (methods returns only enabled countries)
            - change usages in `BillingAddressFormType`, `DeliveryAddressFormType` and `PersonalInfoFormType` in your implementation

>             - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`

Reviewer: ```suggestion
            - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`
            - fix all usages of method `Country::getName` if exist, as it now needs proper locale as an argument
```

            - use new methods `getAllEnabledOnDomain` and `getAllEnabledOnCurrentDomain` (methods returns only enabled countries)
            - change usages in `BillingAddressFormType`, `DeliveryAddressFormType` and `PersonalInfoFormType` in your implementation
            - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`

>             - fix all usages of method `Country::getName` if exist, as it now needs proper locale as an argument

Reviewer 2: ```suggestion
            - if you use `Country::getName()` in your code, fix all the usages of the method as it now needs proper locale as an argument
```

            - use new methods `getAllEnabledOnDomain` and `getAllEnabledOnCurrentDomain` (methods returns only enabled countries)
            - change usages in `BillingAddressFormType`, `DeliveryAddressFormType` and `PersonalInfoFormType` in your implementation

>             - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`

Reviewer: ```suggestion
- fix all usages of method `Country::getName` if exist as it now needs proper locale as an argument
```
what about like this.

>             - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`

Reviewer: yep, no usage of it was in the demoshop but if someone used it, (s)he should fix it based on changes

    - if you have extended `CountryGridFactory`, revise your changes because class changed its namespace
    - if you have extended `CountryFormType`, revise your changes – new fields are available
    - if you have extended `CountryController` revise your changes – `new` and `edit` actions were added

>     - constant `CountryDataFixture::COUNTRY_CZECH_REPUBLIC_1` was renamed to `CountryDataFixture::COUNTRY_CZECH_REPUBLIC`

Reviewer 2: ```suggestion
    - rename `CountryDataFixture::COUNTRY_CZECH_REPUBLIC_1` constant to `CountryDataFixture::COUNTRY_CZECH_REPUBLIC`
```
